### ISSUE TITLE
Misleading notification in publication format metadata form

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#1293]
![selection_045](https://cloud.githubusercontent.com/assets/2306629/13949829/457a0f08-f01f-11e5-88ef-6c812d314e6e.png)

1. It's the Catalog tab where they check the box to create a catalog entry.
2. The conditions for that notification need to be checked. In addition to the catalog entry checkbox, publication formats need to be approved one-by-one in the Publication Formats grid. The notification should be removed or reworded if it's not sensitive to these dual requirements.

**Comment by asmecher** [comment]
Wording adjusted.

**Comment by asmecher** [comment]
The notification used to be relevant to the publication format's metadata modal because there was an approval checkbox there, but we moved that into the Publication Formats grid.

It's intended to alleviate confusion caused by publication formats potentially being complete, but the monograph's listing in the catalog being held up because the monograph itself needs approval.

For 1.2, I've moved the notification to the top of the Production stage, so that someone working with publication format approvals will have a shot at seeing it. That's something that could be improved, but let's return to it after 1.2 is released.

The new wording is:
> The monograph will not be listed in the catalog until it has been approved. To approve catalog metadata, click on the Catalog Entry above and find the Catalog tab.
